_studio/mfx_lib/shared/src/mfx_vpx_dec_common.cpp

```diff
@@ -212,23 +212,15 @@ namespace MFX_VPX_Utility
                 p_out->mfx.FrameInfo.FrameRateExtD = p_in->mfx.FrameInfo.FrameRateExtD;
             }
 
-            if ((p_in->mfx.FrameInfo.AspectRatioW || p_in->mfx.FrameInfo.AspectRatioH) &&
-                (!p_in->mfx.FrameInfo.AspectRatioW || !p_in->mfx.FrameInfo.AspectRatioH))
+            if ((p_in->mfx.FrameInfo.AspectRatioW == 0 && p_in->mfx.FrameInfo.AspectRatioH == 0) ||
+                (p_in->mfx.FrameInfo.AspectRatioW != 0 && p_in->mfx.FrameInfo.AspectRatioH != 0))
```

Comment:
Simplify if() with (p_in->mfx.FrameInfo.AspectRatioW == 0) == (p_in->mfx.FrameInfo.AspectRatioH == 0),
and similar below. Same as in #1079.

Reply:
This will make code smaller but less readable. I'd better leave it as is.

Reply:
Less? Disagree. The same "both equal to 0 or not".